schemas/com.amazon.aws.cloudfront/wd_access_log/jsonschema/1-0-6

```diff
@@ -0,0 +1,112 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a AWS CloudFront web distribution access log event. Version released 24 January 2018",
+  "self": {
+    "vendor": "com.amazon.aws.cloudfront",
+    "name": "wd_access_log",
+    "format": "jsonschema",
+    "version": "1-0-6"
+  },
+
+  "type": "object",
+  "properties": {
+    "dateTime": {
+      "type": "string",
+      "format": "date-time"
+    },
+    "xEdgeLocation": {
+      "type": ["string", "null"],
+      "maxLength": 32
+    },
+    "scBytes": {
+      "type": ["number", "null"]
+    },
+    "cIp": {
+      "type": ["string", "null"],
+      "maxLength": 45
+    },
+    "csMethod": {
+      "type": ["string", "null"],
+      "maxLength": 3
+    },
+    "csHost": {
+      "type": ["string", "null"],
+      "maxLength": 2000
+    },
+    "csUriStem": {
+      "type": ["string", "null"],
+      "maxLength": 8192
+    },
+    "scStatus": {
+      "type": ["string", "null"],
+      "maxLength": 3
+    },
+    "csReferer": {
+      "type": ["string", "null"],
+      "maxLength": 8192
+    },
+    "csUserAgent": {
+      "type": ["string", "null"],
+      "maxLength": 2000
+    },
+    "csUriQuery": {
+      "type": ["string", "null"],
+      "maxLength": 8192
+    },
+    "csCookie": {
+      "type": ["string", "null"],
+      "maxLength": 4096
+    },
+    "xEdgeResultType": {
+      "type": ["string", "null"],
+      "maxLength": 32
+    },
+    "xEdgeRequestId": {
+      "type": ["string", "null"],
+      "maxLength": 2000
+    },
+    "xHostHeader": {
+      "type": ["string", "null"],
+      "maxLength": 2000
+    },
+    "csProtocol": {
+      "enum": ["http", "https", null]
+    },
+    "csBytes": {
+      "type": ["number", "null"]
+    },
+    "timeTaken": {
+      "type": ["number", "null"]
+    },
+    "xForwardedFor": {
+      "type": ["string", "null"],
+      "maxLength": 45
+    },
+    "sslProtocol": {
+      "type": ["string", "null"],
+      "maxLength": 32
+    },
+    "sslCipher": {
+      "type": ["string", "null"],
+      "maxLength": 64
+    },
+    "xEdgeResponseResultType": {
+      "type": ["string", "null"],
+      "maxLength": 32
+    },
+    "csProtocolVersion": {
+      "type": ["string", "null"],
+      "maxLength": 8
+    },
+    "fleStatus": {
+      "type": ["string", "null"],
+      "maxLength": 64
+    },
+    "fleEncryptedFields": {
```

Comment:
Shouldn't it be integer?  From [docs](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/AccessLogs.html) it looks like it is a natural number

Reply:
problem is that it can be "-" too

Reply:
Okay, makes sense.